src/collector.ts

```diff
@@ -105,9 +105,33 @@ class NaiveGomodParser {
     dependencies: Array<IDependency>;
 
     static parseDependencies(contents:string, goImports: Set<string>): Array<IDependency> {
+        let replaceDependencies = new Map();
         let goModDeps = contents.split("\n").reduce((dependencies, line, index) => {
-            // Ignore "replace" lines
-            if (!line.includes("=>")) {
+            if (line.includes("=>")) {
+                // skip any text after '//'
+                if (line.includes("//")) {
+                    line = line.split("//")[0];
+                }
+                // split the replace statements by '=>'
+                const parts: Array<string> = line.replace('replace', '').replace('(', '').replace(')', '').trim().split(' => ');
+                const replaceTo: Array<string> = (parts[0] || '').trim().split(' ');
```

Comment:
Line number 117, 118, 121 can all be avoided if we don't find a valid version in replaced module.
Suggest to move all this inside if block after line no. 124.